book/src/offline-signing/durable-nonce.md

```diff
@@ -0,0 +1,201 @@
+# Durable Transaction Nonces
+
+Durable transaction nonces are a mechanism for getting around the typical
+short lifetime of a transaction's [`recent_blockhash`](../transaction.md#recent-blockhash).
+They are implemented as a Solana Program, the mechanics of which can be read
+about in the [proposal](../implemented-proposals/durable-tx-nonces.md).
+
+## Known Issues
+
+### Fee Theft Opportunity
+
+The durable nonce implementation contains a vulernability which allows for fees
+to be stolen by a transaction using the feature under certain conditions. To
+achieve this, the transaction must have failed with an instruction error, which
+rolls back the step that advanced the stored nonce, allowing the transaction to
+be replayed and fees charged. This can be repeated until the stored nonce is
+successfully advanced.
+
+#### - Mitigation
+
+To minimize loss of funds, a low-balance account should always be used to pay
+fees on a durable nonce transaction.
+
+If a transaction using the durable nonce feature fails with an instruction error,
+a new transaction, which only advances the nonce must be submitted to the network
+immediately.
+
+#### - Issue Tracking
+
+This issue is being actively addressed, progress can be followed on
+[Github](https://github.com/solana-labs/solana/issues/7443).
+
+## Usage
+
+The following are instructions for using the durable transaction nonce feature
+with Solana's CLI tools.
+
+Additionally, authority over a nonce account can be assigned to another entity.
+This enables the creation of more complex account ownership arrangements and
+derived account addresses not associated with a keypair. The
+`--nonce-authority <AUTHORITY_KEYPAIR>` argument is used to specify this
+authority and is supported by the following commands
+* `create-nonce-account`
+* `new-nonce`
+* `withdraw-from-nonce-account`
+* `authorize-nonce-account`
+
+### Nonce Account Creation
+
+The durable transaction nonce feature uses an account to store the next nonce
+value.  Durable nonce accounts must be [rent-exempt](../implemented-proposals/rent.md#two-tiered-rent-regime),
+so need to carry the minimum balance to acheive this.
+
+A nonce account is created by first generating a new keypair
+
+```bash
+solana-keygen new -o <NONCE_KEYPAIR>
+```
+
+{% hint style="info" %}
+To keep the keypair entirely offline, use the [Paper Wallet](../paper-wallet/README.md)
+keypair generation [instructions](../paper-wallet/usage.md#seed-phrase-generation.md)
+instead
+{% endhint %}
+
+Then create the account on chain
+
+```bash
+solana create-nonce-account <NONCE_KEYPAIR> <AMOUNT>
```

Comment:
I think maybe you can just squash the commands `<WITH ARGS>` into the examples. `api-reference.cli.md` should show all the args explicitly, and my suspicion is that this is going to look pretty redundant rendered in the book.

Reply:
Ok. So rather than the usage here, just the examples?

Reply:
That's my inclination